classes/dt-porch-settings.php

```diff
@@ -102,7 +102,7 @@ private static function get_translations( $campaign_id ) {
         return DT_Campaign_Languages::get_translations( $campaign_id );
     }
 
-    public static function update_values( $updates, $campaign_id = null ) {
+    public static function update_values( $updates, $campaign_id = null, $states = [] ) {
```

Comment:
@kodinkat I merged this in, but please let's keep update_values as it is.
What might need to be done here is a direct post update
DT_Posts::update_post

Reply:
@corsacca ok, cool..!